base/cpu/openmpi4.1.0-ubuntu20.04/release-notes.md

```diff
@@ -10,7 +23,7 @@
 -   [CVE-2022-1664](https://ubuntu.com/security/notices/USN-5464-1) 
 -   [CVE-2022-1304](https://ubuntu.com/security/notices/USN-5464-1)  
 
-- released images to mitigate libxml2,PCRE, OpenLDAP dpkg and E2fsprogs Vulnerabilities.
+- released images to mitigate libxml2, PCRE, OpenLDAP dpkg and E2fsprogs Vulnerabilities.
```

Comment:
Check this why you have added all these here ?dont overwrite the old values add new values